tasks.py

```diff
@@ -0,0 +1,56 @@
+import platform
```

Comment:
looks like a rebase went wrong? we deleted this file already from master. and I see some modifications that look unrelated to this PR.

Reply:
It's an extra file I use locally to run tests on specific environments. I removed it.
As for the other modifications, they should be related

e.g : to make it more general and work across different DBs -  I had to change our queries from this

```
SELECT {column}
FROM {table}
GROUP BY 1
ORDER BY 1;
```

to this something like this

```
SELECT {column} as name
FROM {table}
GROUP BY name
ORDER BY name;
```